malaria24/ona/tests/test_tasks.py

```diff
@@ -174,6 +175,177 @@ def test_compile_and_send_digest_email(self):
             mis.email_address]))
         self.assertEqual(len(mail.outbox), 4)
 
+    @responses.activate
+    def test_national_digest_week_variable(self):
+        self.mk_actor(role=MIS,
+                      email_address='manager@example.org')
+        Facility.objects.create(facility_code='342315',
+                                facility_name='Facility 1',
+                                province='Limpopo',
+                                district=u'Example1')
+        self.mk_actor(role=MANAGER_DISTRICT,
+                      email_address='manager@example.org',
+                      facility_code='342315')
+        self.mk_actor(role=MANAGER_PROVINCIAL,
+                      email_address='m2@example.org',
+                      facility_code='342315')
+        self.mk_actor(role=MANAGER_NATIONAL,
+                      email_address='m3@example.org',
+                      facility_code='342315')
+        ehp1 = self.mk_ehp(name='EHP1', email_address='ehp1@example.org')
+        ehp2 = self.mk_ehp(name='EHP2', email_address='ehp2@example.org')
+
+        for i in range(10):
+            case = self.mk_case(gender='female', facility_code='342315')
+            case.date_of_birth = datetime.today().strftime("%y%m%d")
+            case.ehps.add(ehp1)
+            case.ehps.add(ehp2)
+            case.save()
+            case.digest = None
```

Comment:
Because this is added after the `.save()`, this change won't actually be committed to the database. But since it's not in the database, and isn't making a difference, we can probably remove it? (and same for the other cases and tests)

Reply:
are you referring to the case.digest?

Reply:
Yes, I'm referring to the `case.digest = None` line

Reply:
Okay, thanks